examples/sdk/helm-template/go.mod

```diff
@@ -0,0 +1,73 @@
+module helm-template
+
+go 1.20
```

Comment:
looks quite good, just one thing that our troubleshoot go.mod is using go 1.19. Should we keep this sync with it https://github.com/replicatedhq/troubleshoot/blob/f0efbf658a02a36cbb0db3e706a09c808e7f54c8/go.mod#L3C1-L3C8

Reply:
given that this is an example application, I'm comfortable with it running a current Go rather than limiting it to the project's version (which we should update anyway).